listener/listener_config.toml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# --------------------------------------------------
+# Logging -- configuration of service logging
+# --------------------------------------------------
+[Logging]
+LogLevel = "INFO"
+LogFile  = "__screen__"
+
+# -------------------------------------------------------------------
+# Listener -- configuration of Avalon Listener
+# -------------------------------------------------------------------
+
+[Listener]
+bind = "http://avalon-listener:1947"
```

Comment:
Ideally we will set bind or remote_storage_url to localhost in config file and for docker will use the command line argument which will take the preference. So that it will pass in the stand-alone mode no need to pass anything.

Reply:
ok